docs/settings.rst

```diff
@@ -49,6 +49,9 @@ DJANGO_AWS_S3_CUSTOM_DOMAIN             AWS_S3_CUSTOM_DOMAIN        n/a
 DJANGO_AWS_S3_MAX_MEMORY_SIZE           AWS_S3_MAX_MEMORY_SIZE      n/a                                            100_000_000
 DJANGO_GCP_STORAGE_BUCKET_NAME          GS_BUCKET_NAME              n/a                                            raises error
 GOOGLE_APPLICATION_CREDENTIALS          n/a                         n/a                                            raises error
+DJANGO_AZURE_STORAGE_ACCOUNT_KEY        AZURE_ACCOUNT_KEY           n/a                                            raises error
+DJANGO_AZURE_STORAGE_ACCOUNT_NAME       AZURE_ACCOUNT_NAME          n/a                                            raises error
```

Comment:
Shall we just call them `DJANGO_AZURE_ACCOUNT_KEY` and `DJANGO_AZURE_ACCOUNT_NAME`?

Reply:
I thought ACCOUNT_NAME and ACCOUNT_KEY was a bit ambiguous, but I agree that it makes sense to align with what `django-storages` uses.